src/server/scripts/Northrend/VioletHold/boss_lavanthor.cpp

```diff
@@ -21,10 +21,18 @@
 
 enum Spells
 {
-    SPELL_CAUTERIZING_FLAMES                    = 59466, // Only in heroic
+    SPELL_CAUTERIZING_FLAMES                    = 59466, //Only in heroic
```

Comment:
Strictly speaking, removing this single space is not needed, unless there are other comments like this one in the same list.